main.go

```diff
@@ -262,25 +291,31 @@ func (m *Miner) resultLoop() error {
 			case common.ZONE_CTX:
 				log.Println(color.Ize(color.Blue, "ZONE block  : "), header.NumberArray(), header.Hash())
 			}
-			// Send to whichever nodes should be aware of this block
-			if order <= common.ZONE_CTX {
-				m.sendMinedHeader(common.ZONE_CTX, header)
-			}
-			if order <= common.REGION_CTX {
-				m.sendMinedHeader(common.REGION_CTX, header)
-			}
-			if order <= common.PRIME_CTX {
-				m.sendMinedHeader(common.PRIME_CTX, header)
-			}
 		}
 	}
 }
 
 // SendMinedHeader sends the mined block to its mining client with the transactions, uncles, and receipts.
 func (m *Miner) sendMinedHeader(ctx int, header *types.Header) {
-	err := m.client(ctx).ReceiveMinedHeader(context.Background(), header)
-	if err != nil {
-		fmt.Println("error submitting block: ", err)
+	retryDelay := 1 // Start retry at 1 second
+	for {
+		header_req, err := jsonrpc.MakeRequest(0, "quai_receiveMinedHeader", quaiclient.RPCMarshalHeader(header))
+		if err != nil {
+			log.Fatalf("Could not send mined header: %v", err)
+		}
+
+		err = m.sliceClients[ctx].SendTCPRequest(*header_req)
```

Comment:
Since we're only sending blocks to the proxy and backdooring the use of `sliceClients` this PR should just remove `sliceClients` entirely in favor of a proxyConnection.

Reply:
That's of course if we do not want to have optionality to mine directly on the node or the proxy.

Reply:
I agree with wizeguyy. For local testing its unnecessary to run proxy every time we want to test something. Ideally I would like a flag in config to select what I am doing.